docs/content/operations/rule-configuration.md

```diff
@@ -126,6 +129,65 @@ Period drop rules are of the form:
 
 The interval of a segment will be compared against the specified period. The period is from some time in the past to the current time. The rule matches if the period contains the interval.
 
+Broadcast Rules
+---------------
+
+Broadcast rules indicate how segments of different data sources should be co-located in historical nodes. 
+Once a broadcast rule is configured for a data source, all segments of the data source are broadcasted to the servers holding _any segments_ of the co-located data sources.
+
+### Forever Broadcast Rule
+
+Forever broadcast rules are of the form:
+
+```json
+{
+  "type" : "broadcastToDataSourcesForever",
+  "colocatedDataSources" : [ "target_source1", "target_source2" ]
+}
+```
+
+* `type` - this should always be "broadcastToDataSourcesForever"
+* `colocatedDataSources` - A JSON List containing data source names to be co-located. `null` and empty list means broadcasting to every node in the cluster.
+
+### Interval Broadcast Rule
+
+Interval broadcast rules are of the form:
+
+```json
+{
+  "type" : "broadcastToDataSourcesByInterval",
+  "colocatedDataSources" : [ "target_source1", "target_source2" ],
+  "interval" : "2012-01-01/2013-01-01"
+}
+```
+
+* `type` - this should always be "broadcastToDataSourcesByInterval"
+* `colocatedDataSources` - A JSON List containing data source names to be co-located. `null` and empty list means broadcasting to every node in the cluster.
+* `interval` - A JSON Object representing ISO-8601 Periods. Only the segments of the interval will be broadcasted.
+
+### Period Broadcast Rule
+
+Period broadcast rules are of the form:
+
+```json
+{
+  "type" : "broadcastToDataSourcesByPeriod",
+  "colocatedDataSources" : [ "target_source1", "target_source2" ],
+  "period" : "P1M"
+}
+```
+
+* `type` - this should always be "broadcastToDataSourcesByPeriod"
+* `colocatedDataSources` - A JSON List containing data source names to be co-located. `null` and empty list means broadcasting to every node in the cluster.
+* `period` - A JSON Object representing ISO-8601 Periods
+
+The interval of a segment will be compared against the specified period. The period is from some time in the past to the current time. The rule matches if the period contains the interval.
+
+<div class="note caution">
+broadcastToDataSources rules don't guarantee that segments of the data sources are always co-located.
```

Comment:
This comment would be better if it explained why. Maybe adding "because segments for the colocated data sources are not loaded together atomically".